framework/components/simple_node_set/node_set.go

```diff
@@ -128,6 +131,8 @@ func sharedDBSetup(in *Input, bcOut *blockchain.Output) (*Output, error) {
 			if in.NodeSpecs[overrideIdx].Node.TestConfigOverrides != "" {
 				net = in.NodeSpecs[overrideIdx].Node.TestConfigOverrides
 			}
+			nodeName = fmt.Sprintf("node%d", i)
+			nodeWithNodeSetPrefixName := fmt.Sprintf("%s-%s", in.Name, nodeName)
```

Comment:
what if node has a custom name? I guess it will be ignored? if so, remove it from node input?

Reply:
IMO there is no need to name the nodes, only node sets, which clutters the tests, happy to fix it in the next releases if we figure out it's otherwise.